dm/syncer/data_validator.go

```diff
@@ -77,12 +76,11 @@ type tableChange struct {
 type rowChange struct {
 	table      *validateTableInfo
 	key        string
-	pkValues   []string
+	pkValues   []string // todo: might be removed in later pr
```

Comment:
Currently, the `pkValues` is used to construct the query condition whereas the `key` is the concatenation of `pkValues`. Would it be better to deprecate the `key` field but concat it every time we gonna use it? Otherwise how can we construct the `Cond` struct?🤔

Reply:
`pkValues` can also be recreated from `rowChange.data`+`table.pkIndices`, but key is used more often